src/js/angular/clustermanagement/controllers.js

```diff
@@ -81,6 +82,39 @@ function ClusterManagementCtrl($scope, $http, $q, toastr, $repositories, $modal,
             });
     };
 
+    $scope.showDeleteDialog = () => {
+        const modalInstance = $modal.open({
+            templateUrl: 'js/angular/clustermanagement/templates/modal/cluster-delete-dialog.html',
+            controller: 'DeleteClusterCtrl'
+        });
+
+        modalInstance.result.then(function (forceDelete) {
+            const loaderMessage = $translate.instant('cluster_management.delete_cluster_dialog.loader_message');
+            $scope.setLoader(true, loaderMessage);
+            ClusterRestService.deleteCluster(!!forceDelete)
+                .success((data) => {
+                    const partialDelete = Object.values(data).every((node) => node === 'DELETED');
```

Comment:
This is confusing. Can you invert this partialDelete, because now you enter in the success branch when partialDelete === true. Right? Shouldn't this be oposite?

Reply:
Yes, my  bad. It should be the other way around. I changed the name of the varialble to `allNodesDeleted` and now it makes sense.